oada-core/docker-compose.yml

```diff
@@ -130,7 +130,7 @@ services:
       - ./oada-srvc-docker-config.js:/oada-srvc-docker-config.js
     environment:
       - NODE_ENV=${NODE_ENV:-development}
-      - DEBUG=${DEBUG:-""}
+      - DEBUG=${DEBUG:-*:error,*:warn,*:info}
```

Comment:
I'm not sure we want to have all this on by default?

Reply:
I think microservices should always print those messages by default or you will not notice them unless you turn them on.

Reply:
Each microservice should print one-line info per event

Reply:
I would say error should definitely be on by default. But In production I'm not sure you need the info messages. maybe warnings.

Reply:
I have changed most of `info` calls to `trace` but maybe it's still excessive. But I would like to see at least one `info` message per request like what most HTTP servers print to an access log

Reply:
Sounds good